content/php-client/resources.md

```diff
@@ -0,0 +1 @@
+# Resources
```

Comment:
Needed to concat every files into the menu "resources". Or I need to put this logic in the gulp task but not a big fan.